modules/sno-du-managing-networking.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-managing-networking_{context}"]
+= Managing networking
```

Comment:
Specifying cluster installation parameters
```suggestion
= Specifying cluster installation parameters
```
It may make sense to combine the ClusterDeployment, AgentClusterInstall and InfraEnv CRs into this same section since they are all specifying different installation attributes. The current breakdown of sections feels arbitrary.

Reply:
changed.